docs/faq_tips_tricks.rst

```diff
@@ -0,0 +1,31 @@
+.. include:: links.rst
+
+FAQ, Tips, and Tricks
+------------------------------
+
+**Should I run quality control of my images before running ``FMRIPREP``?**
+
+Yes. You should as well before any other processing/analysis.
+
+Oftentimes (more often than we would like), images have fatal artifacts and problems. 
+Those would not pass any QC check.
+Some other times, QC checks will flag some images that we should carefully track throughout processing.
+
+When using publicly available datasets, an additional concern is that images may have gone through some kind of preprocessing (see next question).
+
+
+**What if I find some images have undergone some pre-processing already (e.g. my T1w image is already skull-stripped)?**
+
+These images imply an unknown level of preprocessing (e.g. was it already bias-field corrected?), which makes it difficult to decide on best-practices for further processing.
+Hence, supporting such images was considered very low priority for ``FMRIPREP``.
```

Comment:
I think we've generally moved to this format. We should try to be consistent across pages, so maybe have a look before accepting this one.

```suggestion
Hence, supporting such images was considered very low priority for fMRIPrep.
```